app/src/components/Media/MediaList.tsx

```diff
@@ -19,6 +20,7 @@ interface IMediaListProps {
   listName?: string;
   items: IMedia[];
   showFilter?: boolean;
+  genres?: IGenre[];
```

Comment:
Also, change the type of genres prop into IGenre[] | undefined so what so ever so that it does not cause any IDE error on 122th line.

_Type 'IGenre[] | undefined' is not assignable to type 'IGenre[]'.
  Type 'undefined' is not assignable to type 'IGenre[]'.ts(2322)_

Reply:
@ctuanle éc file ni k hiểu sao git nó k push lên nời, trách chi cái props listType k có nên preview khác với local

ok now ✌️